ITISFoundation/osparc-simcore
# Quick fixes

## What do these changes do?


A couple of ``iframe`` bugs fixed:
* Maximizing an ``iframe`` now works.
* We can now resize a SidePanel over an ``iframe``

Other bugs:
* File picker bug fixed. #741 

## Test it please
* Try maximizing minimizing all kind of iframes, navigate away from them, check that everything works as it should.
* Try resizing a SidePanel over an ``iframe`` and see that it works. Try with all kind of iframes. Before it was set a z-index of 1000 for every PersistentIframe. Now that setting is removed, do you think that could cause any issue?

## Checklist
